com.unity.render-pipelines.high-definition/Editor/RenderPipeline/Settings/SerializedScalableSetting.cs

```diff
@@ -134,52 +131,27 @@ ScalableSettingSchema schema
         /// <param name="position">The rect to use to draw the GUI.</param>
         /// <param name="subLabels">The labels for each sub value field.</param>
         /// <param name="values">The current values of the fields.</param>
-        static void MultiField<T>(Rect position, GUIContent[] subLabels, T[] values)
+        static void MultiField<T>(GUIContent[] subLabels, T[] values)
```

Comment:
Well MultiIntField, MultiFloatField, MultiPropertyField are in UnityEditor a way to draw several item on the same line.

As this is changed, can you find a different name for this method to prevent any ambiguity ?

Reply:
Also the documentation need to be updated (removed the position argument)

Reply:
I've used the other multifields that we already have.